<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TermBuilder</class>
 <widget class="QMainWindow" name="TermBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SHATRNS | DTE Assistant | Lane Community College</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_InstNum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Institution Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_InstNum">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_FICE">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FICE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FICE">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_StartRow">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Row</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_StartRow">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_EndRow">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End Row</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_EndRow">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHome">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Return Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButtonHome</tabstop>
  <tabstop>lineEdit_InstNum</tabstop>
  <tabstop>lineEdit_FICE</tabstop>
  <tabstop>lineEdit_StartRow</tabstop>
  <tabstop>lineEdit_EndRow</tabstop>
  <tabstop>pushButtonGo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
